src/index.js

```diff
@@ -45,7 +45,7 @@ export const createContext = (defaultValue) => {
 export const useContextSelector = (context, selector) => {
   const listeners = context[CONTEXT_LISTENERS];
   if (!listeners) {
-    throw new Error('useContextSelector requires special context');
+    throw new Error(process.env.NODE_ENV !== 'production' ? 'useContextSelector requires special context' : '');
```

Comment:
If we were to go more, how about this?
```suggestion
    throw (process.env.NODE_ENV !== 'production' ? new Error('useContextSelector requires special context') : new Error());
```
Would this save two bytes?